### PR TITLE
MGMT-19068: Adjust the periodicity of e2e RESTAPI jobs

### PR DESCRIPTION
**Pull request by danmanor**
Should be -

 
N: 5 times a week

N-1: 2 times a week

N-2: 2 times a week

N-3: 1 time a week

N-4: 1 time a week

### CODE REVIEW

**Review by adriengentil — APPROVED** [review]

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml [hunk 1]
```diff
@@ -174,19 +174,8 @@ tests:
         OPENSHIFT_VERSION=4.14
         OPENSHIFT_INSTALL_RELEASE_IMAGE=""
     workflow: assisted-baremetal
-- as: e2e-metal-assisted-4-12-periodic
```
Comment (adriengentil) [inline ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:177]: 4.12 is still in extended support, we may want to keep it? https://access.redhat.com/support/policy/updates/openshift

Comment (danmanor) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:177]: I thought just leaving the presubmit, but yes you are right

Comment (danmanor) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:177]: Fixed, thank you

**Review by adriengentil — APPROVED** [review]

**Pull request merged**